src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceContractTests.cs

```diff
@@ -336,8 +334,7 @@ public static void NetTcp_NoSecurity_Echo_RoundTrips_String_StreamedResponse()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(713)]
-    public static void NetTcp_NoSecurity_Echo_RoundTrips_String_Streamed()
+    public static void NetTcp_NoSecurity_Streamed_RoundTrips_String()
```

Comment:
Do you think this makes sense as a [Theory] based test? it looks like the things we're changing are just the TransferMode.